src/elements/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Library from './../Library/Library';
+import Search from './../Search/Search';
+import Details from './../Details/Details';
+import Footer from './../Footer/Footer';
+import './Homepage.css';
+
+class Homepage extends React.Component {
+    state = {
+        detailsId: null
+    }
+
+    render() {
+        return (
+            <div className="homepage">
+                {this.state.detailsId
```

Comment:
why not make usage of props, instead of a state here?

Reply:
I wanted this to be my top stateful component I do not need any state above that (eventually I could move it to redux but we are not there yet so I assumed state is my best shot here), is my thinking wrong?